src/Chamber.sol

```diff
@@ -106,6 +106,30 @@ contract Chamber is IChamber, Common {
 
         if (!onVoterList) revert invalidApproval("TokenId not on voter list");
 
+        if ((proposals[_proposalId].approvals+1)==3){
+            // Gas estimation for the first proposal
+            uint256 baseGas = 110539;
```

Comment:
Where are you getting this value from? How can you be certain this value is going to be valid for a wide range of possible transactions?

Reply:
Please read this 
https://github.com/loreum-org/chamber/pull/47#issuecomment-1905242615